mdn/kuma
# Show "Thank You" header on Payments Landing page if subscriber

Changes: 
- Add `ThankYouSubheader` component. We have the same header used on two pages, so this avoids duplicating content
- Move shared strings to strings file. This made testing easier, since I didn't have to re-type strings. 
- CSS edits to fix a few issues in the header
- Add tests

To test: 
- As a subscriber, visit http://localhost.org:8000/en-US/payments/. Should see the Thank You header and member number: 
<img width="1151" alt="Screen Shot 2020-04-06 at 9 46 32 PM" src="https://user-images.githubusercontent.com/650/78621273-3c250f80-7850-11ea-9750-c6db1de14fa6.png">

- Cancel your subscription, and refresh /payments. Should see the signup header and form:
<img width="1196" alt="Screen Shot 2020-04-06 at 9 39 36 PM" src="https://user-images.githubusercontent.com/650/78621109-c1f48b00-784f-11ea-9a2f-76722d583e22.png">

- Go to http://localhost.org:8000/en-US/payments/thank-you, should see Thank You header and member number, like the first step:
<img width="1151" alt="Screen Shot 2020-04-06 at 9 46 32 PM" src="https://user-images.githubusercontent.com/650/78621273-3c250f80-7850-11ea-9750-c6db1de14fa6.png">

- Sign out and visit http://localhost.org:8000/en-US/payments/thank-you, should see Thank You header with no member number: 
<img width="1006" alt="Screen Shot 2020-04-06 at 9 48 26 PM" src="https://user-images.githubusercontent.com/650/78621355-68409080-7850-11ea-8987-2a8e7d6bab18.png">

Closes #6757
